lib/misc.coffee

```diff
@@ -1,3 +1,11 @@
+locker = (_this, params..., fun) ->
```

Comment:
`_this` is reserved by coffeescript.

I really like the idea of the locker (i agree that the current implementation was a bit messy). But I would propose to implement the locker as a class. Furthermore, it would be nice if you catch errors here.

```
class Locker
  constructor: ()->
    @lock = false

  tryAction: (f)->
    ..
```

The advantage being, that you don't put a property on some element (in this case y-richtext). Instead, you initialize Locker once in y-richtext
```
locker = new misc.Locker()
locker.tryAction ()=> # you can bind to this here
  ...
```